src/github.com/matrix-org/dendrite/clientapi/routing/register.go

```diff
@@ -50,9 +50,36 @@ const (
 	sessionIDLength   = 24
 )
 
+// sessionsDict represents every sessions' completed flow stages.
```

Comment:
this might be better phrased as "sessionsDict keeps track of completed auth stages for each session"